build/scripts/build.ps1

```diff
@@ -25,7 +25,7 @@ function Print-Usage() {
 }
 
 function Run-Build() {
-    $buildArgs = "/v:m /m"
+    $buildArgs = "/v:m /m /flp:v=diag"
```

Comment:
This slows down our build considerably.  Why do we need to make this a part of the dev workflow?